src/main/java/frc/robot/controls/XboxDriverConfig.java

```diff
@@ -0,0 +1,5 @@
+package frc.robot.controls;
+
+public class XboxDriverConfig {
+    
+}
```

Comment:
If you are creating this class, put something in it. You can copy and paste most things from GameControllerDriverConfig. Also, I think the game controllers might be xbox controllers.

Reply:
done